test/cql-pytest/test_compaction_strategy_validation.py

```diff
@@ -6,38 +6,48 @@
 # Tests for compaction strategy validation
```

Comment:
I addressed the points from the commit message under the issue (https://github.com/scylladb/scylladb/issues/16027#issuecomment-1808044240). I would leave it as is. Changing of scylla's `bucket_low` message is worth considering, but that's out of scope for this PR.

Reply:
Ok, so I'm leaving my partial patch for this test as I did it. It means that even after my patch, this test will fail on Cassandra. It's not a disaster - nobody runs it on Cassandra anyway (it's not part of our CI), and hopefully another day you'll look into these issues. None of this is urgent at all.